### ISSUE TITLE
Hook example

### ISSUE THREAD

**Issue by jacekpmlr** [pqina/filepond#151]
Can you give an example of how to use `beforeRemoveFile` hook? I'd like to use it for showing confirmation modal.

**Comment by rikschennink** [comment]
This works when removing a file while `instantUpload` is set to `false`. I just noticed it doesn't trigger on revert when `instantUpload` is set to `true`, so I'm fixing that now.

```js
FilePond.create({
    beforeRemoveFile: () => {
        return confirm('sure?');
    }
});
```

**Comment by rikschennink** [comment]
Just published version `3.2.3` which fixes the issue.
